minecraft_servers/melonblocks/manifest.json

```diff
@@ -0,0 +1,41 @@
+{
+  "server_name": "melonblocks",
+  "nice_name": "MelonBlocks",
+  "direct_ip": "melonblocks.net",
+  "server_wildcards": [
+    "%.melonblocks.net"
```

Comment:
Doesnt look like a real wildcard
<img width="820" alt="image" src="https://github.com/user-attachments/assets/28f0b055-9b8f-4950-b05d-3649845610e0">

Reply:
Could you check again?

Reply:
It should be changed now.

Reply:
looks good 👍